virtualizarr/tests/test_xarray.py

```diff
@@ -271,6 +272,19 @@ def test_combine_by_coords(self, netcdf4_files):
         assert combined_vds.xindexes["time"].to_pandas_index().is_monotonic_increasing
 
 
+
+
+
+
+
+pytest.importorskip("s3fs")
+@pytest.mark.parametrize("filetype", ['netcdf4', None], ids=["netcdf4 filetype", "None filetype"])
+@pytest.mark.parametrize("indexes", [None, {}], ids=["None index", "empty dict index"])
+def test_anon_read_s3(filetype, indexes):
+    #TODO: Switch away from this s3 url after minIO is implemented.
+    fpath = 's3://carbonplan-share/virtualizarr/local.nc'
+    assert open_virtual_dataset(fpath, filetype=filetype, indexes=indexes)
```

Comment:
Can we assert a little more about the structure here? Maybe at least checking that certain variables / indexes are present? Because currently this would pass even if it returned just a non-empty string (somehow).

Reply:
Good call!